packages/base/src/panelview/components/layers.tsx

```diff
@@ -410,6 +410,7 @@ function LayerComponent(props: ILayerProps): JSX.Element {
       onDragOver={Private.onDragOver}
       onDragEnd={Private.onDragEnd}
       data-id={layerId}
+      style={{ textDecoration: layer.failed ? 'line-through' : 'none' }}
```

Comment:
What do you think about displaying a warning icon :warning: and when you hover it you can see the error message(s)?